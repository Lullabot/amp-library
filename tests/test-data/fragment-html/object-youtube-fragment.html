<object width="512" height="308">
	<param name="movie" value="https://www.youtube.com/embed/MnR9AVs6Q_c">
	<param name="allowFullScreen" value="true">
	<param name="allowscriptaccess" value="always">
	<embed src="https://www.youtube.com/embed/MnR9AVs6Q_c" type="application/x-shockwave-flash" allowscriptaccess="always" allowfullscreen="true" width="512" height="308">
</object>