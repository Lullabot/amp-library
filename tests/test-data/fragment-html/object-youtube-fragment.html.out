<amp-youtube data-videoid="MnR9AVs6Q_c" layout="responsive" height="308" width="512"></amp-youtube>

ORIGINAL HTML
---------------
Line 1: <object width="512" height="308">
Line 2: 	<param name="movie" value="https://www.youtube.com/embed/MnR9AVs6Q_c">
Line 3: 	<param name="allowFullScreen" value="true">
Line 4: 	<param name="allowscriptaccess" value="always">
Line 5: 	<embed src="https://www.youtube.com/embed/MnR9AVs6Q_c" type="application/x-shockwave-flash" allowscriptaccess="always" allowfullscreen="true" width="512" height="308">
Line 6: </object>


Transformations made from HTML tags to AMP custom tags
-------------------------------------------------------

<object width="512" height="308"> at line 1
 ACTION TAKEN: object youtube object code was converted to the amp-youtube tag.


AMP-HTML Validation Issues and Fixes
-------------------------------------
PASS

COMPONENT NAMES WITH JS PATH
------------------------------
'amp-youtube', include path 'https://cdn.ampproject.org/v0/amp-youtube-0.1.js'

